digitalocean-1-click/image.pkr.hcl

```diff
@@ -12,7 +12,7 @@ source "digitalocean" "x" {
   api_token    = "${var.token}"
   image        = "ubuntu-20-04-x64"
   region       = "sfo3"
-  size         = "s-1vcpu-1gb"
+  size         = "s-1vcpu-2gb"
```

Comment:
I had to change the image size used to create the digital ocean image because EdgeDB was getting killed for OOM violations. This change means that users would no longer be able to chose an image with less than 2GB of ram.

Reply:
We should be able to work around by temporarily adding a swap file.

Reply:
Adding a swap file worked for building the image but EdgeDB still gets killed form memory usage setting a password which is the first step in the deployment guide.

Reply:
I change the memory back to 1GB and added a permanent swap file.